protected/models/Module.php

```diff
@@ -891,4 +891,31 @@ public function getLecturesDataProvider() {
         ));
     }
 
+    /**
+     * Creates new lecture.
+     * @param array $params must incude fields 'titleUa', 'titleRu', 'titleEn', 'order';
+     */
+
+    public function addLecture($params) {
+
+        $teacher = Teacher::model()->find('user_id=:user', array(':user' => Yii::app()->user->getId()));
+
+        $lecture = Lecture::model()->addNewLesson(
+            $this->module_ID,
+            $params['titleUa'],
+            $params['titleRu'],
+            $params['titleEn'],
+            $teacher->teacher_id
+        );
+
+        $this->lesson_count = $params['order'];
+
```

Comment:
this looks a little bit strange. Why lesson count became equal the order? We have a very old bug related to the lesson's order